shell/src/main/java/tachyon/shell/command/PinCommand.java

```diff
@@ -46,4 +46,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "pin <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Pins the given file or folder in memory (works recursively for a directory). Pinned files are never evicted from memory";
```

Comment:
`Pins the given file or directory in memory (works recursively for directories). Pinned files are never evicted from memory.`